### ISSUE TITLE
How to initialize weights of classifier at finetuning stage?

### ISSUE THREAD

**Issue by hiyijian** [seasonSH/DocFace#3]
Dear @seasonSH ,
  Would you like to teach us how to initialize weights of classifier at finetuning stage, which is not mentioned in your paper?

**Comment by hiyijian** [comment]
BTW. Can you point out DWI related code lines or blocks. I am not very familar with TF. Sorry for that

**Comment by seasonSH** [comment]
I have just uploaded the contents of DocFace+, including DIAM-Softmax. You can find the function in "src/tflib.py" (named diam_softmax). The weights are initialized in such a way that they should be distributed uniformly on the sphere. In practice, because I normalize all the weights onto the sphere every step (including first one), it is sufficient to just initialize them with a Xavier or Normal distribution.

**Comment by hiyijian** [comment]
Thanks.
